src/views/dashboard/Home.tsx

```diff
@@ -2,20 +2,32 @@ import { useSession } from "next-auth/react";
 import { useState } from "react";
 import { toast } from "react-hot-toast";
 
+import EmptySyncsIcon from "public/svg/empty-syncs.svg";
+
 import { Button } from "../../components/common/button/Button";
+import { Empty } from "../../components/common/empty/Empty";
 import { Profile } from "../../components/dashboard/profile/Profile";
+import { SyncsList } from "../../components/dashboard/sync/SyncsList";
 import { Tile } from "../../components/dashboard/tile/Tile";
 import { AddFeedModal } from "../../components/modal/feed/AddFeedModal";
 import { DASHBOARD_CARDS } from "../../utils/consts";
 import { trpc } from "../../utils/trpc";
 
 import type { CreateFeedInput } from "../../utils/validation";
+import type { Sync } from "@prisma/client";
 import type { TRPCError } from "@trpc/server";
 
 export const HomeView = () => {
   const utils = trpc.useContext();
   const { data } = useSession();
   const [isAddModalOpen, setIsAddModalOpen] = useState(false);
+  const [syncs, setSyncs] = useState({
```

Comment:
I think we can use lazy initialization or move initial value of state outstide component.

Reply:
What is the reasons behind it?

Reply:
On each re-render initial value is created. (re-renders caused by `useQuery` hook (render 1: loading, render 2: success), I did't see any server side fetching to avoid loading).

Reply:
Yeah, you're right, I think I should refactor this view a bit :)